app/vminsert/prompush/push.go

```diff
@@ -57,7 +57,7 @@ func push(ctx *common.InsertCtx, tss []prompbmarshal.TimeSeries) {
 			label := &ts.Labels[j]
 			ctx.AddLabel(label.Name, label.Value)
 		}
-		if !ctx.TryPrepareLabels(false) {
+		if !ctx.TryPrepareLabels(true) {
```

Comment:
I guess, it can be changed in the same way as other insert protocols doing and add `hasRelabeling` variable:

```
	hasRelabeling := relabel.HasRelabeling()
```